### PR TITLE
Fix minor mistakes

### PR DESCRIPTION
**Pull request by projanen**

### CODE REVIEW

File: documentation/manual/detailedTopics/production/Production.md [hunk 1]
```diff
@@ -77,12 +77,12 @@ Please consult the [documentation](http://www.scala-sbt.org/sbt-native-packager)
 
 ### Build a server distribution
 
-The sbt-native-packager plugins provides a number archetypes.  The one that Play uses by default is called the Java server archetype, which enables the following features:
+The sbt-native-packager plugins provides a number of archetypes.  The one that Play uses by default is called the Java server archetype, which enables the following features:
```
Comment (benmccann) [inline documentation/manual/detailedTopics/production/Production.md]: this should be "plugins provide". not sure if you want to fix that one too while you're at it

File: documentation/manual/detailedTopics/production/Production.md [hunk 2]
```diff
@@ -77,12 +77,12 @@ Please consult the [documentation](http://www.scala-sbt.org/sbt-native-packager)
...
 
 * System V or Upstart startup scripts
 * [Default folders](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/my-first-project.html#default-mappings)
 
-A full documentation can be found in the [documentation](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html).
+The full documentation can be found in [documentation](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html).
```
Comment (benmccann) [inline documentation/manual/detailedTopics/production/Production.md]: "the full documentation can be found in documentation" sounds a bit funny. What do you think about changing to:

[The full server archetype documentation](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/index.html) is available in [the sbt-native-packager documentation](http://www.scala-sbt.org/sbt-native-packager/)

**Comment by cchantep** [comment]
I guess it needs to be updated according @benmccann comments before merge.

**Comment by dotta** [comment]
@projanen I'm closing this PR as there has been no update in 2 months. Please, feel free to reopen when it's ready.

**Pull request closed without merging**